media-gfx/rawtherapee/Manifest

```diff
@@ -1,3 +1,3 @@
-DIST rawtherapee-5.1.tar.xz 10785516 BLAKE2B 5c8ae3f90c4ef4bc352b95761a5a2fd94e1541f6c7cec749c0985c95b85a7fd7cd5e96d439639513715b463c318d0f7b000cf48582b5391b944fc7a6ecaa3be8 SHA512 6a881fe24a4109c7db39cb8bee500eb2f9357c479ddb32b3c9b7987aa5d12e4f10083533b5657a0e0f0713fe4ebc58e0dcf204eb8d5c5b656f3e7867a775ab19
```

Comment:
Please split this into two commits, first one adding the new version and the other removing old.

Reply:
@mgorny 
done, I've reverted and re-done in the same branch, please advise if a different pull request something has to be changed